influent-java/src/main/java/influent/forward/ForwardServer.java

```diff
@@ -45,6 +47,14 @@
     private boolean keepAliveEnabled = true;
     private boolean tcpNoDelayEnabled = true;
     private int workerPoolSize = 0;
+    private String protocol = "TCP";
+    private String tlsVersion = "TLS";
```

Comment:
We prefer `String[] tlsVersions` to `String tlsVersion`.
If multiple versions are available, Influent ensure backward compatibility when upgrading TLS version.